tests/e2e/support/objects/runtime/application.ts

```diff
@@ -34,14 +34,10 @@ export class Application {
           resp.url().endsWith('notifications?format=json') &&
           resp.status() === 200 &&
           resp.request().method() === 'GET'
-      ),
-      this.#page.reload()
+      )
```

Comment:
I wouldn't recommend it. Because `this.#page.reload()` there to cut out the notifications waiting time.

Reply:
See https://github.com/owncloud/web/pull/9433